registry/modules/auth-basic/tests/authBasic.Controller.test.ts

```diff
@@ -0,0 +1,113 @@
+import { describe, it, expect, vi, beforeEach, Mock } from "vitest";
```

Comment:
Great test class. Only comment is that usually tests are located next to the file they are testing. e.g. this test class would ideally be named 
```
registry/modules/auth-basic/controllers/authBasic.test.ts
```

However, this brings up the question, do we want to install the test classes alongside the module?

Reply:
That is why I decided to have them in a separate folder so we don't clone them in the main project. However, I think it is worth discussing if we need the tests in the main project, I think it might be very useful for the developers and we would pre offering even more "complete" API Modules.